### DISCUSSION TITLE
Transportation Factors Boundary - Well to Wheel or Tank to Wheel

### DISCUSSION THREAD

**Discussion by smcnichols94** [USEPA/supply-chain-factors#87] [Q&A] [answered]
Hello,

Can you please confirm what the boundary of the transportation related emissions factors in the EPA supply chain factors is? Do these factors include well to wheel emissions or just tank to wheel?

Thanks,
Susan

**Comment by bl-young** [comment, marked as answer]
All of the factors include full supply chains as well as direct emissions. So for transportation sectors this would be most similar to well to wheel.

**Reply by smcnichols94** [reply]
Got it - thank you!
